onnxscript/function_libs/torch_lib/graph_building_test.py

```diff
@@ -118,6 +118,18 @@ def outer(x: FLOAT[1, 2, 3]):
         expected = outer.to_model_proto()
         onnxscript.testing.assert_isomorphic(traced, expected)
 
+    def test_adding_different_tensors_using_same_initializer_name_raises(self):
```

Comment:
```suggestion
class TestTorchScriptGraph(unittest.TestCase):
    def test_add_initializer_raises_when_the_same_name_used_for_different_tensors(self):
    
```

Create a test class for the graph class and update the same to something along this line?